managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerTest.java

```diff
@@ -2922,6 +2922,23 @@ public void testManagedLedgerRollOverIfFull() throws Exception {
         Assert.assertEquals(ledger.getTotalSize(), 0);
     }
 
+    @Test
+    public void testLedgerReachMaximumRolloverTime() throws Exception {
+        ManagedLedgerConfig config = new ManagedLedgerConfig();
+        config.setRetentionTime(1, TimeUnit.SECONDS);
+        config.setMaxEntriesPerLedger(2);
+        config.setMinimumRolloverTime(1, TimeUnit.MILLISECONDS);
+        config.setMaximumRolloverTime(1, TimeUnit.SECONDS);
+
+        ManagedLedger ml = factory.open("ledger-reach-maximum-rollover-time", config);
+        Position position1 = ml.addEntry("test".getBytes());
+
+        Thread.sleep(1000);
```

Comment:
Why not use Awaitility

Reply:
This test is used for mocking a sense that the ledger reaches the maximum rollover time threshold.

Reply:
what happens on a slow machine ?
will the test fail ?

what are you testing here ? is there a better way to test what you want ?

Reply:
I changed the test. PTAL @315157973 @eolivelli